CONTRIBUTING.md

```diff
@@ -27,9 +37,75 @@ again.
 3. Please include unit tests (and integration tests if applicable) for all new code.
 4. Make sure all existing tests pass.
    * In `jib-core`, run `./gradlew clean goJF build integrationTest`
-   * In `jib-gradle-plugin`, run `./gradlew clean goJF build` *
-   * In `jib-maven-plugin`, run `./mvnw clean fmt:format verify` *
+   * In `jib-gradle-plugin`, run `./gradlew clean goJF build`
+   * In `jib-maven-plugin`, run `./mvnw clean fmt:format verify -Pintegration-tests`
 5. Associate the change with an existing issue or file a [new issue](../../issues).
 6. Create a pull request!
 
 \* *Note that you will not be able to run the integration tests for `jib-gradle-plugin` or `jib-maven-plugin` because those push to our integration-testing GCP project. If you would like to run integration tests, change all uses of the `jib-integration-testing` project to your own GCP project and run `./gradlew integrationTest` for `jib-gradle-plugin` and `./mvnw verify -Pintegration-tests` for `jib-maven-plugin`.*
+
+## Debugging the Jib Maven Plugin (`jib-maven-plugin`)
+
+### Build and use a local snapshot 
+
+To use a local build of the `jib-maven-plugin`:
+
+  1. build and install `jib-core` into your local `~/.m2/repository`
```

Comment:
`jib-core` doesn't actually need to be installed since `jib-maven-plugin` directly uses the files in the relative directory.